<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Manual_MultitagInfo</class>
 <widget class="QDialog" name="Manual_MultitagInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>881</width>
    <height>744</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MultiTag Information</string>
  </property>
  <widget class="QPushButton" name="pushButton_done">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>710</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_previousPage">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>710</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Prev. Page</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_nextPage">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>710</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Next Page</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_page2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>781</width>
     <height>411</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:700;&quot;&gt;MultiTag Parameters, Continued&lt;/span&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:700;&quot;&gt;High Priority:&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;If this value is set to TRUE, then this MultiTag will be considered before Tags and other MultiTags with normal priority.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;It is recommended that you set MultiTags to normal priority unless you absolutely need a MultiTag to be chosen before anything else.&lt;/span&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:700;&quot;&gt;Requires Fulfillment:&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Since a MultiTag can be linked to multiple Tags, it is possible that not all Linked Tags can schedule a Meal.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;If you require that absolutely all of a MultiTags's Linked Tags schedule their requested number of Meals, set this to YES.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:700; font-style:italic;&quot;&gt;Note, if not all Linked Tags can schedule their Requested Meals, nothing will be scheduled and this MultiTag will be ignored.&lt;/span&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;But, if you're happy with at least 1 Meal in total being scheduled, then set this to NO.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;(For info on plan generation, see the manual page about Meal Plan generation, under &amp;quot;How do I use this program?&amp;quot;)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_pageNumber">
   <property name="geometry">
    <rect>
     <x>405</x>
     <y>670</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:700;&quot;&gt;Page 1/5&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_page1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>691</width>
     <height>601</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:700;&quot;&gt;MultiTag Parameters&lt;/span&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:700;&quot;&gt;Name:&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Name of a MultiTag, it should describe what you are categorizing in a few words.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;For example: &amp;quot;Four-course Meal&amp;quot;, &amp;quot;Breakfast, Lunch, Dinner&amp;quot;, &amp;quot;Randomized Sandwich&amp;quot; (See page 3 for an explanation.)&lt;/span&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:700;&quot;&gt;Description:&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;A brief description about what sort of Tags are linked to this MultiTag.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;For example: &amp;quot;Meals for each member of the family.&amp;quot;, &amp;quot;Meals for the whole day.&amp;quot;, &amp;quot;Components of a sandwich&amp;quot;&lt;/span&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:700;&quot;&gt;Enabled Days:&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;A MultiTag can be enabled or disabled for every day of the week. It is ignored on a disabled day.&lt;/span&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:700;&quot;&gt;Linked Tags:&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;MultiTags have no purpose unless linked to Tags.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Meals rely on one or more assigned Tags to be scheduled.&lt;/span&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;MultiTags are groupings of Tags, and allows for multiple Meals to be scheduled on a single day.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;MultiTags also allow for the use of complex rules, as both MultiTags and their Linked Tags have Enabled Days.&lt;/span&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Every Tag linked to a MultiTag can request one or more Meals&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Normally, only one Meal is scheduled from a Tag on a day of the week.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;However, requesting more than one Meal makes the program schedule more Meals from the same Tag on the same day.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_page3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>841</width>
     <height>561</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:700;&quot;&gt;Advanced Usage: Sandwich Randomizer&lt;/span&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Meals can be as small or large as you want them to be.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;While it is advised you make Meals as complete as possible, advanced usage of MultiTags may require small Meals, such as the following:&lt;/span&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Suppose that you created a Meal called &amp;quot;Slice of white bread&amp;quot;.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;This is a poor meal in the traditional sense, but it is a component of a sandwich, which will become useful as you will see..&lt;/span&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Now suppose you also created multiple Meals for many types of bread, multiple types of bread (Slice of sourdough, Slice of whole wheat, etc.).&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Next, you create Meals for the main component of a sandwich: &amp;quot;Roast beef&amp;quot;, &amp;quot;Smoked turkey&amp;quot;, etc.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;You also create Meals for the toppings: &amp;quot;Mustard&amp;quot;, &amp;quot;Ketchup&amp;quot;, &amp;quot;Pickles&amp;quot;, etc.&lt;/span&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Then, you create a Tag called &amp;quot;Sandwich Buns&amp;quot;, and assign the slices of bread to this Tag.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Then another Tag called &amp;quot;Sandwich Core&amp;quot; and assign &amp;quot;Roast beef&amp;quot;, &amp;quot;Smoked turkey&amp;quot;, etc., to this Tag.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Yet again, you create a Tag called &amp;quot;Sandwich Toppings&amp;quot;, and assign it to the toppings you created as Meals.&lt;/span&gt;&lt;br/&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Finally, you create a MultiTag called &amp;quot;Random Sandwich&amp;quot;, and link it to the newly created Tags.&lt;/span&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;You set the number of requested meals for &amp;quot;Sandwich Buns&amp;quot; to 2 (for a top and bottom), while the other Linked Tags request 1 Meal each.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;And also set the Tags to Depends on MultiTag: YES, so that they only get chosen for Meal scheduling if &amp;quot;Random Sandwhich&amp;quot; is chosen.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Since you want a complete sandwich, you set Requires Fulfillment to YES, so that incomplete sandwiches are not scheduled.&lt;/span&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Now, if the MultiTag &amp;quot;Random Sandwhich&amp;quot; is chosen, then all of the components of a sandwich will be scheduled on the same day of the week.&lt;/span&gt;&lt;br/&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;You will have effectively created a custom sandwich creator! (for more practical examples, see the next page.)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_page4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>841</width>
     <height>501</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:700;&quot;&gt;Advanced Usage: Part 2&lt;/span&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Hopefully the previous page illustrated the utility of using MultiTags, although a sandwich randomizer is not the most practical use of one.&lt;/span&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;In reality, you might want to use MultiTags to plan multiple Meals for each member of a household.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;In this case, you will assign such Meals to one or more Tags, then link the Tag(s) to a MultiTag.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;When linking the Tag(s), you must specify how many Meals you want to schedule from the Tag being linked.&lt;/span&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;A Linked Tag's Enabled Days will always be respected.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;So if a Linked Tag is disabled on every day of the week, but the MultiTag is enabled on all days, then program will &lt;/span&gt;&lt;span style=&quot; font-size:10pt; font-style:italic;&quot;&gt;never&lt;/span&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt; schedule any meals from the Linked Tag.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;This is because the Linked Tag is effectively disabled, despite the MultiTag being enabled on every day.&lt;/span&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Because of this, it is important to be mindful of the restrictions set for a Tag.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Only limit the Enabled Days of a MultiTag when you are absolutely sure you want to restrict it.&lt;/span&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;If a Tag is enabled on every day, but the MultiTag that is linked to it is disabled on every day, then the result is no Tags will &lt;/span&gt;&lt;span style=&quot; font-size:10pt; font-style:italic;&quot;&gt;ever&lt;/span&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt; be chosen from the MultiTag.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;This is because the MultiTag, which contains Linked Tags, is effectively disabled.&lt;/span&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;So if the MultiTag is not enabled, the state of its Linked Tags do not matter at all, they will never be chosen.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
